ArticlesApp/ArticlesApp/ArticlesViewController.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+class ArticlesViewController: UIViewController {
+
+    @IBOutlet weak var loadingView: UIView!
+    @IBOutlet var articlesTableView: UITableView!
```

Comment:
`articlesTableView` can also be `weak` here, since you already have a reference for it coming from Interface Builder.